### ISSUE TITLE
Feedback regarding new player experience with the platform

### ISSUE THREAD

**Issue by Gridster33** [Yellow-Dog-Man/Resonite-Issues#2101]
### Is your feature request related to a problem? Please describe.

As a new player i experienced 2 major disconnects with the platform.
1) The inventory system is not tutorialized in any way. to learn about how it works, it requires other players to teach it. this could be acceptable if the tutorial was a lesser part of the experience, but i think it would be a good idea to tutorialize the basics of it at the very least for new players.
2)  The desktop environment and the vr environment have a large flaw in the inability to rebind keys, or at least not having alternate default keybinds. This is the kind of thing that is going to cause non-creator type players to bounce of the game because they are just trying to have the game feel good to interact with and if they can't do that, it's problematic, when they have already sunk large quantities of hours in other platforms. 

### Describe the solution you'd like

For problem 1, i propose an additional tutorial to be added to the tutorial world to teach players the inventory system
for problem 2, an alternate default keybind option, or a keybinding remapping option. 

### Describe alternatives you've considered

Feature 1:
an extra tutorial that needs to be done before the ability to go to your home world where you take some item, (preferably something important like a the resonite essentials folder) from the world and you save it to your inventory. and this way players who are new get introduced to that concept early and without other players.
Feature 2:
for the bare minimum, it would be nice to have an alternate control scheme that more closely mirrors VRC's default controls. although it would be appreciated to have the ability to bind/unbind keys to actions, like being able to just make a button on the controller not do anything in game so we can have it handle external things like playspace mover, OVR, or any other element. 

### Requesters

ZetaKane in game
zetake on discord

**Comment by Frooxius** [comment]
Thank you for the feedback!

1. For the inventory, we might add some additional tutorial somewhere. We do plan to rework the Inventory UI though to be much more intuitive, so we might not put too much effort into tutorializing the current one: https://github.com/Yellow-Dog-Man/Resonite-Issues/issues/1119

2. For the ability to rebind keys, we already have a thread here: https://github.com/Yellow-Dog-Man/Resonite-Issues/issues/23

I'm not sure how closely you'd be able to mimic VRC default controls, since this is a different platform, with different sets of actions, so things won't map 1:1

We'd need to know a few more details on what particular aspects of the controls trip people up and that would make them more familiar to people. I usually use VRC myself, so I'm not sure what people expect there.

**Comment by Gridster33** [comment]
I think one of the big things is menu and context menu specifically in regards to the default. With them being flipped, it's quite uncomfortable to get used as a new player. 
They are used super often, and in vrc they are flipped the other way around input wise. Most of the other inputs were fine, but that was something that stood out very obviously as an annoyance as a new player.

**Comment by shiftyscales** [comment]
In the case of Resonite, the context menu is typically a much more commonly used menu than the dash because it has features such undo, redo, and the locomotion selector which don't exist on other platforms, @Gridster33. So we wouldn't be likely to change the default- but as noted, users later could change their preferred action bindings with #23 as needed/desired.

**Comment by shiftyscales** [comment]
Further to point 1- there are additional tutorials / content planned to address the learning curve, e.g. #109 / #539.

We also wish to make access to existing documentation easier via #1123.

Thank you for your feedback. Please feel free to comment on any of the above linked issues if you have any specific points of feedback you wish to raise on any of those issues.
